docs/code-of-conduct.md

```diff
@@ -0,0 +1,89 @@
+# Zulip Code of Conduct
+
+## Our Commitment
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers are committed to making participation in our
+project and our community a harassment-free experience for everyone, regardless
+of age, body size, disability, ethnicity, gender identity and expression, level
+of experience, nationality, personal appearance, race, religion, or sexual
+identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being friendly and patient
+* Being respectful of differing viewpoints and experiences, and trying to
+  understand other people's points of view
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Assuming good faith on the part of other community members
+* Respecting the time and attention of others, in particular by doing research
```

Comment:
I think it's important to discuss the issue of wasting other people's time, but I agree that version isn't ideal.  How about:
```
* Respecting the time and attention of others, in particular:                          
  * Including tracebacks, any changes you've made, and other relevant                  
    details in bug/error reports.                                                      
  * Asking broadly relevant questions in public forums, rather than                    
  sending private messages to individuals, so that the first available                 
  expert can answer, and the answers are available to the whole                        
  community.                                                                           
```